unstructured/chunking/title.py

```diff
@@ -218,8 +250,9 @@ def wrapper(*args: _P.args, **kwargs: _P.kwargs) -> List[Element]:
                 elements = chunk_by_title(
                     elements,
                     multipage_sections=params.get("multipage_sections", True),
-                    combine_under_n_chars=params.get("combine_under_n_chars", 500),
-                    new_after_n_chars=params.get("new_after_n_chars", 1500),
+                    combine_text_under_n_chars=params.get("combine_text_under_n_chars", 500),
+                    new_after_n_chars=params.get("new_after_n_chars", 1000),
+                    max_characters=params.get("max_characters", 1500),
```

Comment:
maybe set the default values like 500 as constants? Here unlike in the function API it is not exposed to the caller so the default value is hardwired in and hidden, which can make it difficult to maintain. Also we have those constants being reused in multiple places so it is better if we use a constant variable so that it is easier to maintain the code as well.

Reply:
Yes definitely! will do :)